sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -653,5 +652,81 @@ def Transport(host, connect_timeout=None, ssl=False, **kwargs):
     Given a few parameters from the Connection constructor,
     select and create a subclass of _AbstractTransport.
     """
+    transport_type = kwargs.pop('transport_type')
+    if transport_type == TransportType.AmqpOverWebsocket:
+        transport = WebSocketTransport
     transport = SSLTransport if ssl else TCPTransport
```

Comment:
wouldn't the transport be overwritten by line 658 `transport = SSLTransport if ssl else TCPTransport` if `transport_type` is passed?